### PR TITLE
[ LAKESIDE ] FLS-499: Sort a group of courses as per the hardcoded order #2274

### PR DESCRIPTION
**Pull request by OPersian**
**Description:** Describe in a couple of sentences what this PR adds

**Youtrack:** https://youtrack.raccoongang.com/issue/FLS-499

### CODE REVIEW

File: lms/templates/index.html [hunk 1]
```diff
@@ -179,6 +179,83 @@ <h2 class="header-search-facets">${_('Refine Your Search')}</h2>
...
+              allCourses = firstCoursesHtml.concat(remainingCoursesHtml);
+              el.innerHTML = allCourses.join("");
+            } else {
+              console.info("Didn't sort a certain group of courses, and it's fine.")
+            }
+
+          }
+        }
+
+        setTimeout(reorderCourses, 3000);
```
Comment (OPersian) [inline lms/templates/index.html]: 3000 is for really slow connection

setTimeout is to wait for all other scripts to execute (".courses-listing"'s are created with scripts)

Comment (GlugovGrGlib) [reply lms/templates/index.html]: let's do 1000, it will be enough, as all users are from the North America

File: lms/templates/index.html [hunk 2]
```diff
@@ -179,6 +179,83 @@ <h2 class="header-search-facets">${_('Refine Your Search')}</h2>
          if(getParameterByName('next')) {
```
Comment (GlugovGrGlib) [inline lms/templates/index.html:179]: interesting https://stackoverflow.com/questions/4584373/difference-between-window-load-and-document-ready-functions

File: lms/templates/index.html (hunk 1 above)
Comment (OPersian) [reply lms/templates/index.html]: addressed

**Comment by OPersian** [comment]
Gonna reject this one too :woman_shrugging: in favor of a edx-search patch

**Pull request closed without merging**